src/discussions/posts/data/api.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable import/prefer-default-export */
-import { ensureConfig, getConfig } from '@edx/frontend-platform';
+import snakeCase from 'lodash.snakecase';
+import { ensureConfig, getConfig, snakeCaseObject } from '@edx/frontend-platform';
```

Comment:
May I know the reason when we have `snakeCaseObject ` then why we are adding lodash package again and using both of these in same file?

Reply:
The reason for this is that snakeCaseObject only converts the **keys** of an object from snake case to camel case, however in this case we also need to modify some values.

Reply:
I see, I think we should have that utility available from frontend-platform like other utilities, but I searched there is none.
I can discuss with this team further for now it makes sense.

Reply:
If it helps, `frontend-platform` uses the same function: https://github.com/edx/frontend-platform/blob/1e062a3579842c5adcbf8f8f5953a1f51f24e994/src/utils.js#L6-L7

That's where I looked, so I copied over the dependency. I think what we should do is to just reexport that from frontend-platform so it's available to import from there. That way a future version of the platform can replace lodash with a different implementation.

Reply:
I have discussed this within the team and they think you should be fine using "loadash" as a dependency here.  frontend-platform doesn’t bundle its own dependencies, so you should only get one copy of the lodash functions.

Reply:
marking it resolved no change needed.